_includes/metas-ogp.html

```diff
@@ -0,0 +1,18 @@
+
+    <title>{% if page.title %}{{ page.title }} | {{ site.title }}{% else %}{{ site.title }}{% endif %}</title>
```

Comment:
Let's delete these two lines since title is already part of the default layout file

Reply:
Would you mind if move them from the default layout to the metas-ogp ? In this way the metas-ogp is exactly the same for all the 'headers for crawlers' that I've pushed to KubeVirt and Metal³